ovos-i2csound

```diff
@@ -98,6 +98,10 @@ main() {
     if [[ ${detection_results[MARK1]} == true ]] ; then
         echo "Installing and configuring WM8731 based sound HAT"
         dtoverlay proto-codec
+        dtoverlay pi3-disable-bt
+        dtoverlay pi3-miniuart-bt
```

Comment:
ovos@mark1-dev:~ $ sudo dtoverlay pi3-disable-bt
DTOVERLAY[warn]: overlay 'pi3-disable-bt' has been renamed 'disable-bt'

Reply:
To be fair, this was tested on a Mark 1 with a Pi 4 inside.

Reply:
I can rename to `disable-bt` instead, what other changes?  These are what @JarbasAl needed to get his faceplate mouth to work

Reply:
ovos@mark1-dev:~ $ sudo dtoverlay pi3-miniuart-bt
DTOVERLAY[warn]: overlay 'pi3-miniuart-bt' has been renamed 'miniuart-bt'
* Failed to apply overlay '2_miniuart-bt' (kernel)

FWIW:

ovos@mark1-dev:~ $ uname -srm
Linux 6.1.21-v8+ aarch64

Reply:
same in a pi3

Reply:
If this is what was needed, and we're not sure when the redirect was introduced, probably best to leave it for now? I don't want to break compat with Pi3 Mk1 devices accidentally